### ISSUE TITLE
Performance drop when using Distributed Data Parallel

### ISSUE THREAD

**Issue by NarsimhaChilkuri** [burchim/EfficientConformer#10]
Hello,

I am noticing that using DistributedDataParallel, although significantly faster, results in a higher loss compared to DataParallel training. Is this expected? 

I am working with the small Conformer RNN-T model, and also note that I have only trained the models for a few epochs (<< 250 epochs).

Thank you

**Comment by NarsimhaChilkuri** [comment]
I think the difference was due to differences in batch_size (and hence the total the number of gradient descent steps)
